src/Model/Item.php

```diff
@@ -2670,6 +2671,16 @@ private static function tagDeliver($uid, $item_id)
 			}
 		}
 
+		if (!$mention) {
+			$tags = Tag::getByURIId($item_id, [Tag::MENTION, Tag::EXCLUSIVE_MENTION]);
```

Comment:
`$item_id` doesn't contain the `uri-id` expected by `Tag::getByURIId`, I believe it should be something like
```suggestion
			$tags = Tag::getByURIId($item['uri-id'], [Tag::MENTION, Tag::EXCLUSIVE_MENTION]);
```

Reply:
Yes, it should.